Jenkinsfile

```diff
@@ -154,7 +154,7 @@ def RunIntegrationTest(k, v) {
                               export PATH="/tmp/rook-tests-scripts-helm/linux-amd64:$PATH" \
                                   KUBECONFIG=$HOME/admin.conf
                               kubectl config view
-                              _output/tests/linux_amd64/integration -test.v -test.timeout 2400s --host_type '''+"${k}"+''' --helm /tmp/rook-tests-scripts-helm/linux-amd64/helm 2>&1 | tee _output/tests/integrationTests.log'''
+                              _output/tests/linux_amd64/integration -test.v -test.timeout 7200s --host_type '''+"${k}"+''' --helm /tmp/rook-tests-scripts-helm/linux-amd64/helm 2>&1 | tee _output/tests/integrationTests.log'''
```

Comment:
the typical time for the integration tests is currently 35 minutes. If it goes over 60 minutes, seems like jenkins should kill it because the tests are surely failing. The longer we wait, the jenkins backlog just has the potential to get longer. Hopefully we can fix the resource issue with jenkins soon...